pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1322,16 +1334,22 @@ func (ctrl *PersistentVolumeController) provisionClaim(claim *v1.PersistentVolum
 	}
 	glog.V(4).Infof("provisionClaim[%s]: started", claimToClaimKey(claim))
 	opName := fmt.Sprintf("provision-%s[%s]", claimToClaimKey(claim), string(claim.UID))
+	startTime := time.Now()
 	ctrl.scheduleOperation(opName, func() error {
-		ctrl.provisionClaimOperation(claim)
-		return nil
+		err := ctrl.provisionClaimOperation(claim)
```

Comment:
You can return volume plugin name from `provisionClaimOperation` so you don't need to look it up below.